RangePicker/tests/RangePicker-specs.js

```diff
@@ -1,105 +1,117 @@
-import {mount} from 'enzyme';
-import {RangePicker, RangePickerBase} from '../RangePicker';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
-const tap = (node) => {
-	node.simulate('mousedown');
-	node.simulate('mouseup');
-};
-const decrement = (slider) => tap(slider.find('Button').last());
-const increment = (slider) => tap(slider.find('Button').first());
+import {RangePicker, RangePickerBase} from '../RangePicker';
 
 describe('RangePicker Specs', () => {
 	test('should render a single child with the current value', () => {
-		const picker = mount(
+		const {getByText} = render(
 			<RangePicker min={-10} max={20} value={10} />
 		);
 
-		const expected = '10';
-		const actual = picker.find('PickerItem').text();
+		const textContent = getByText('10');
```

Comment:
This could be confused with `.textContent` from jest. Could you change it to something else? Maybe `value`?

Reply:
Ok